x/wasm/internal/keeper/test_common.go

```diff
@@ -115,10 +117,12 @@ var TestingStakeParams = stakingtypes.Params{
 type TestKeepers struct {
 	AccountKeeper authkeeper.AccountKeeper
 	StakingKeeper stakingkeeper.Keeper
+	// deprecated use WasmKeeperRef instead
```

Comment:
Why keep both of these? This was my confusion with the setup above.

If needed for some commits to get to a stable state, makes sense. But let's remove it and replace all calls with `WasmKeeperRef` before merging.

Reply:
👍  I tried to not touch too many things here but better go the full mile.